src/bundle/Resources/views/parts/page_title.html.twig

```diff
@@ -5,6 +5,10 @@
             <svg class="ez-icon ez-icon-{{ iconName }}">
                 <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#{{ iconName }}"></use>
             </svg>
+        {% elseif contentType is defined %}
+            <svg class="ez-icon ez-icon-{{ contentType }}">
```

Comment:
Something is wrong here. From a line below `contentType` is an object not string. In the other template you pass `identifier` as a `contentType`.

Reply:
You was right, very good catch, I'm impressed! 😄 
Fixed in: https://github.com/ezsystems/ezplatform-admin-ui/pull/805/commits/92412c2a9ae2715780dd70178e11f1f96c29dee8